components/search/list-renderer/prediction-set.js

```diff
@@ -0,0 +1,43 @@
+// node_modules
+import PropTypes from "prop-types";
+// components/search/list-renderer
+import AlternateAccessions from "../../alternate-accessions";
```

Comment:
You actually get `AlternateAccessions` from `components`, not `components/search/list-renderer`, so it should really go in its own section under a `// components` comment.

Reply:
Thanks. I copied that from auxiliary set, so it looks like the same issue is in that file too. I will fix it here, though.